Source/Initialization/WarpXInitData.cpp

```diff
@@ -131,21 +131,35 @@ WarpX::InitPML ()
 {
     if (do_pml)
     {
+        amrex::IntVect do_pml_Lo_corrected = do_pml_Lo;
+        // amrex::IntVect do_pml_Hi_corrected = do_pml_Hi;
+
+#ifdef WARPX_DIM_RZ
+        do_pml_Lo_corrected[0] = 0; //no PML along z-axis
+#endif
         pml[0].reset(new PML(boxArray(0), DistributionMap(0), &Geom(0), nullptr,
                              pml_ncell, pml_delta, 0,
 #ifdef WARPX_USE_PSATD
                              dt[0], nox_fft, noy_fft, noz_fft, do_nodal,
 #endif
-                             do_dive_cleaning, do_moving_window));
+                             do_dive_cleaning, do_moving_window,
+                             do_pml_Lo_corrected, do_pml_Hi));
         for (int lev = 1; lev <= finest_level; ++lev)
         {
+            amrex::IntVect do_pml_Lo_MR = amrex::IntVect::TheUnitVector();
+#ifdef WARPX_DIM_RZ
+            if ((max_level > 0) && (fine_tag_lo[0]==0.)) { //if the border of the patch matches with the z-axis
```

Comment:
Could you move the above comment into its own line (in order to avoid lines with more than ~80 characters), and slightly rephrase it:
```
// if the edge of the patch is at r=0, do not add PML
if ((max_level > 0) && (fine_tag_lo[0]==0.)) { 
```